lib/brakeman/tracker/config.rb

```diff
@@ -111,6 +111,8 @@ def set_rails_version
         @escape_html = true
         Brakeman.notify "[Notice] Escaping HTML by default"
       end
+
+      check_haml_version
```

Comment:
Kind of a weird place for this...but I don't know if there is a less weird place.